src/bp-members/classes/class-bp-members-admin.php

```diff
@@ -2674,6 +2674,72 @@ public function users_table_process_bulk_type_change() {
 								}
 							}
 						}
+
+						/**
+						 * For add the new profile type roles
+						 **/
+						if ( !$error ) {
+							// Get post id of selected profile type.
+							$post_id = bp_member_type_post_by_type( $new_type );
+							
+							// Get selected profile type role.
+							$selected_member_type_wp_roles = get_post_meta( $post_id, '_bp_member_type_wp_roles', true );
+							$member_type_role    = sanitize_text_field( $selected_member_type_wp_roles[0] );
+							$member_type_role    = ( isset( $member_type_role ) ) ? $member_type_role : '';
```

Comment:
@manishpatolia follow the proper indentaion of code

Reply:
Changed